src/SessionList.ts

```diff
@@ -284,6 +281,10 @@ export class PersistentSessionList extends ASessionList {
       const myworkspaces = data.filter((d) => d.creator === me);
       const otherworkspaces = data.filter((d) => d.creator !== me);
 
+      const publicTitle = i18n.t('tdp:core.SessionList.status') as string;
+      const privateTitle = i18n.t('tdp:core.SessionList.status', {context: 'private'}) as string
+      const unknownText = i18n.t('tdp:core.SessionList.unknown')
```

Comment:
```suggestion
      const unknownText = i18n.t('tdp:core.SessionList.unknown') as string;
```

No `as string` necessary?

Reply:
Thanks for the suggestion , no `as string` necessary when u assign `i18n` result to a variable.